projects/plugins/jetpack/modules/protect.php

```diff
@@ -616,16 +720,17 @@ function kill_login() {
 		 */
 		do_action( 'jpp_kill_login', $ip );
 
-		if( defined( 'XMLRPC_REQUEST' ) && XMLRPC_REQUEST ) {
+		if ( defined( 'XMLRPC_REQUEST' ) && XMLRPC_REQUEST ) {
+			// translators: variable is the IP address that was flagged.
 			$die_string = sprintf( __( 'Your IP (%1$s) has been flagged for potential security violations.', 'jetpack' ), str_replace( 'http://', '', esc_url( 'http://' . $ip ) ) );
 			wp_die(
-				$die_string,
-				__( 'Login Blocked by Jetpack', 'jetpack' ),
-				array ( 'response' => 403 )
+				$die_string, // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- esc_url used when forming string.
+				__( 'Login Blocked by Jetpack', 'jetpack' ), // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
You could escape this one I think.

```suggestion
				esc_html__( 'Login Blocked by Jetpack', 'jetpack' ),
```